ReactiveCocoa/Swift/SignalProducerExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SignalProducerExtensions.swift
+//  ReactiveCocoa
+//
+//  Created by Nate Stedman on 3/22/16.
+//  Copyright © 2016 GitHub. All rights reserved.
+//
+
+import enum Result.NoError
+
+extension SignalProducer {
+	func demoteErrors() -> SignalProducer<Value, NoError> {
+		return flatMapError { _ in SignalProducer<Value, NoError>.empty }
```

Comment:
Yeah. I think we can incorporate some of the extensions [here](https://gist.github.com/NachoSoto/25b0003c1b15d7d11a8d).